client/test/unitTests/Store/UserAccess.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { act, renderHook } from '@testing-library/react';
+import useUserData from 'store/UserAccess';
+import { wrapWithInitialState } from '../testUtils';
+
+describe('UserAccess', () => {
+  const defaultRender = wrapWithInitialState();
+  const testUserName = 'testUser';
+
+  it('should update user correctly with setUser', () => {
```

Comment:
I recommend extention of testUtil. We need something like
```ts
export function renderHookWithInitialState(hook: function) {
  return renderHook(() => hook, {
      wrapper: defaultRender,
    });
}
```

Reply:
Could it make sense to give optional state extension? 
```ts
export function renderHookWithInitialState(hook: function, initState?)
```

This will also invite refactoring of other uses of the wrapWithInitialState() function.